.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 
+
+### PR information
+- [ ] Title of the PR is clear and informative
+- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](../documentation/cleaning-up-commits.md).
+- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
+- [ ] If applicable, the PR references the bug/issue that it fixes.
+- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).
+
+### Quality Swagger
```

Comment:
Resolved. I was trying to do a checklist like "Quality Swagger..." "conforms to the specification", "follows the patterns described...", but that was needlessly confusing. I changed it so each list item is a full sentence.